### ISSUE TITLE
strange training speed for different dataset

### ISSUE THREAD

**Issue by RNGod** [AlexeyAB/darknet#3151]
Hi AlexeyAb, many thanks to your contribution to this repo. So I have 1 yolo model that trained on a huge dataset (9k data), it took 17 seconds to complete 1 iteration whereas another model that trained on small dataset (1k data) took 44 seconds for 1 iteration. Both using the same GPU, same config file (learning rate, input size, batch size.. etc). The only difference is the category of object labelled in the dataset. So, shouldn't the model with 9k data take a longer time to train for 1 iteration than the 1k data model?

**Comment by AlexeyAB** [comment]
@RNGod Hi,

Very high image resolution can reduce speed.

Do you use my repository https://github.com/AlexeyAB/darknet that is compiled with `OPENCV=1` for training?

**Comment by RNGod** [comment]
@AlexeyAB 

Thats the strange part, both dataset has the same dimension as well hence same resolution. yes, I used your repository but its not compiled with OPENCV, its only with GPU=1, CUDNN=1.

**Comment by AlexeyAB** [comment]
@RNGod 
Try to compile it with OPENCV=1 in the Makefile, it will train faster.
And if you train via SSH, then use flag `-dont_show` during training.
